compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -105,6 +105,13 @@ enum class VectorPreProcStrategy {
   None
 };
 
+static llvm::cl::opt<bool> clArmSMETiling(
+    "iree-llvmcpu-arm-sme-tiling",
```

Comment:
very nit: I think '`arm-sme-tiling`' being true/false sounds a little odd, maybe rename to something like:  `iree-llvmcpu-tile-for-arm-sme=true/false` or `iree-llvmcpu-use-arm-sme-tiling=true/false`